tests/installation/accept_license.pm

```diff
@@ -23,21 +23,19 @@ use testapi;
 
 sub run {
     my ($self) = @_;
-    assert_screen 'license-agreement';
-    $self->verify_license_has_to_be_accepted;
-    send_key $cmd{next};
-
-    # workaround bsc#1058503 - license shown twice on s390x
-    # if next screen has a network settings button, everything is fine
-    if (check_var('ARCH', 's390x')) {
-        assert_screen([qw(network-settings-button license-agreement-accepted)]);
-        if (match_has_tag('license-agreement-accepted')) {
-            record_soft_failure 'bsc#1058503 - license shown twice';
-            save_screenshot;
-            send_key $cmd{next};
+    assert_screen([qw(network-settings-button license-agreement-accepted)]);
```

Comment:
I think it's fair to say that this broke *all* sle15 tests but yours.

Reply:
yeah... I should have removed the `-accepted` here...